README.md

```diff
@@ -122,1907 +122,10 @@ const { items } = await lastSucceededRunClient.dataset().listItems();
-
-
-* * *
-
-<a name="WebhookClient+get"></a>
-
-#### [](#WebhookClient+get) `webhookClient.get()` ⇒ <code>Promise.&lt;?Webhook&gt;</code>
-
-https://docs.apify.com/api/v2#/reference/webhooks/webhook-object/get-webhook
-
-
-* * *
-
-<a name="WebhookClient+update"></a>
-
-#### [](#WebhookClient+update) `webhookClient.update(newFields)` ⇒ <code>Promise.&lt;Webhook&gt;</code>
-
-https://docs.apify.com/api/v2#/reference/webhooks/webhook-object/update-webhook
-
-
-| Param | Type |
-| --- | --- |
-| newFields | <code>object</code> |
-
-
-* * *
-
-<a name="WebhookCollectionClient"></a>
-
-### [](#WebhookCollectionClient) WebhookCollectionClient
-
-
-* [WebhookCollectionClient](#WebhookCollectionClient)
-    * [`.create([webhook])`](#WebhookCollectionClient+create) ⇒ <code>Promise.&lt;Webhook&gt;</code>
-    * [`.list([options])`](#WebhookCollectionClient+list) ⇒ [<code>Promise.&lt;PaginationList&gt;</code>](#PaginationList)
-
-
-* * *
-
-<a name="WebhookCollectionClient+create"></a>
-
-#### [](#WebhookCollectionClient+create) `webhookCollectionClient.create([webhook])` ⇒ <code>Promise.&lt;Webhook&gt;</code>
-
-https://docs.apify.com/api/v2#/reference/webhooks/webhook-collection/create-webhook
-
-
-| Param | Type |
-| --- | --- |
-| [webhook] | <code>object</code> |
-
-
-* * *
-
-<a name="WebhookCollectionClient+list"></a>
-
-#### [](#WebhookCollectionClient+list) `webhookCollectionClient.list([options])` ⇒ [<code>Promise.&lt;PaginationList&gt;</code>](#PaginationList)
-
-https://docs.apify.com/api/v2#/reference/webhooks/webhook-collection/get-list-of-webhooks
-
-
-| Param | Type |
-| --- | --- |
-| [options] | <code>object</code> |
-| [options.limit] | <code>number</code> |
-| [options.offset] | <code>number</code> |
-| [options.desc] | <code>boolean</code> |
-
-
-* * *
-
-<a name="WebhookDispatchClient"></a>
-
-### [](#WebhookDispatchClient) WebhookDispatchClient
-
-
-* * *
-
-<a name="WebhookDispatchClient+get"></a>
-
-#### [](#WebhookDispatchClient+get) `webhookDispatchClient.get()` ⇒ <code>Promise.&lt;?WebhookDispatch&gt;</code>
-
-https://docs.apify.com/api/v2#/reference/webhook-dispatches/webhook-dispatch-object/get-webhook-dispatch
-
-
-* * *
-
-<a name="WebhookDispatchCollectionClient"></a>
-
-### [](#WebhookDispatchCollectionClient) WebhookDispatchCollectionClient
-
-
-* * *
-
-<a name="WebhookDispatchCollectionClient+list"></a>
-
-#### [](#WebhookDispatchCollectionClient+list) `webhookDispatchCollectionClient.list([options])` ⇒ [<code>Promise.&lt;PaginationList&gt;</code>](#PaginationList)
-
-https://docs.apify.com/api/v2#/reference/webhook-dispatches/webhook-dispatches-collection/get-list-of-webhook-dispatches
-
-
-| Param | Type |
-| --- | --- |
-| [options] | <code>object</code> |
-| [options.limit] | <code>number</code> |
-| [options.offset] | <code>number</code> |
-| [options.desc] | <code>boolean</code> |
-
-
-* * *
-
+All public classes, methods and their parameters can be inspected in the [API reference](/api).
```

Comment:
moreover this one is wrong, we changed the URL to `reference` (and it is not in the root of the website, so i guess the absolute links will be necessary)